schema/types/base/fact.pb.go

```diff
@@ -0,0 +1,183 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.26.0
+// 	protoc        v3.17.3
+// source: proto/persistence/schema/types/base/v1beta1/fact.proto
+
+package base
+
+import (
+	types "github.com/cosmos/cosmos-sdk/codec/types"
+	_ "github.com/regen-network/cosmos-proto"
+	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
+	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
+	reflect "reflect"
+	sync "sync"
+)
+
+const (
+	// Verify that this generated code is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(20 - protoimpl.MinVersion)
+	// Verify that runtime/protoimpl is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(protoimpl.MaxVersion - 20)
+)
+
+type Fact struct {
+	state         protoimpl.MessageState
+	sizeCache     protoimpl.SizeCache
+	unknownFields protoimpl.UnknownFields
+
+	//  option (cosmos_proto.implements_interface) = "Fact";
+	HashID     *types.Any `protobuf:"bytes,1,opt,name=hashID,proto3" json:"hashID,omitempty"`
```

Comment:
possible to make the members not pointers?

Reply:
Yes, already changed in the new PR